website/source/docs/providers/aws/r/lambda_permission.html.markdown

```diff
@@ -65,7 +65,7 @@ EOF
 resource "aws_lambda_permission" "with_sns" {
   statement_id  = "AllowExecutionFromSNS"
   action        = "lambda:InvokeFunction"
-  function_name = "${aws_lambda_function.my-func.arn}"
+  function_name = "${aws_lambda_function.my-func.function_name}"
```

Comment:
While this is _not_ documented, the API accepts either ARN or name here - happy to accept this change though, as it makes it look cleaner. 👍